styles/RegistrationStatus.css

```
@@ -0,0 +1,487 @@
+  --perplexity-modal-overlay: rgba(0, 0, 0, 0.7);
+}
+
+.dashboard-container {
+  min-height: 100vh;
+  background-color: var(--perplexity-bg);
+  background-image: 
+    radial-gradient(at 0% 0%, rgba(16, 185, 129, 0.03) 0px, transparent 50%),
+    radial-gradient(at 100% 0%, rgba(245, 158, 11, 0.03) 0px, transparent 50%);
+  color: var(--perplexity-text-main);
+  transition: background-color 0.3s ease, color 0.3s ease;
+  font-family: 'Inter', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)